website/src/tabs/transaction/SendCredits.js

```diff
@@ -0,0 +1,108 @@
+import React, {useState} from "react";
+import {Card, Divider, Form, Input, Button} from "antd";
+import {useAleoWASM} from "../../aleo-wasm-hook";
+import {downloadAndStoreFiles, getAllSavedFiles} from '../../db';
+
+export const SendCredits = () => {
+    const [privateKey, setPrivateKey] = useState("APrivateKey1zkp3dQx4WASWYQVWKkq14v3RoQDfY2kbLssUj7iifi1VUQ6");
+    const [toAddress, setToAddress] = useState("aleo184tj0fllfuzqzpmw5jt6l2ptx0avhjxh95u9llcr6ypf6fx3hvrsref0ju");
+    const [amount, setAmount] = useState(50);
+    const [plaintext, setPlaintext] = useState(`{
+      owner: aleo184vuwr5u7u0ha5f5k44067dd2uaqewxx6pe5ltha5pv99wvhfqxqv339h4.private,
+      gates: 1159017656332810u64.private,
+      _nonce: 1635890755607797813652478911794003479783620859881520791852904112255813473142group.public
+    }`);
+    const [transaction, setTransaction] = useState(null);
+    const aleo = useAleoWASM();
+
+    const safeStateUpdate = (update, event) => {
+      try { update(event.target.value) }
+      catch (error) { console.error(error)}
+    }
+
+    const buildTransaction = async () => {
+      try {
+        const pK = aleo.PrivateKey.from_string(privateKey);
+        const add = aleo.Address.from_string(toAddress);
+        const rec = aleo.RecordPlaintext.fromString(plaintext);
+        const transaction = await aleo.TransactionBuilder.build_transfer_full(pK, add, BigInt(amount), rec);
+        setTransaction(transaction);
+      } catch (error) { console.error(error) }
+    }
+
+    const loadFiles = async () => {
```

Comment:
When playing with the UI, hit the "Load Files" button first and then the "Create Transaction" button once that finishes. Files are saved locally so you should only have to do the download once.

File management for arbitrary programs is going to require a lot of thought.